### DISCUSSION TITLE
Which Chrome version supports latest (tip-of-tree) CDP version?

### DISCUSSION THREAD

**Discussion by freb** [go-rod/rod#731] [Q&A] [answered]
I'm trying to test out the [Audits](https://chromedevtools.github.io/devtools-protocol/tot/Audits/) events, but when I call:

~~~go
err := proto.AuditsEnable{}.Call(cdp)
if err != nil {
fmt.Println(err)
}
~~~

I get the error:

~~~
id: 1: {-32601 'Audits.enable' wasn't found }
~~~

So I checked the browser protocol version with

~~~go
proto.BrowserGetVersion{}.Call(cdp)
~~~

which returned `1.3`. I read somewhere that you needed to be running the dev version of chrome, so I launched that version and still have the same result.

Is the issue with rod where I'm not setting the protocol version, or is the issue with chrome (I'm running vChrome/107.0.5300.0)?

**Comment by ysmood** [comment, marked as answer]
Works fine for me, I think only a page can use Audits:

```go
// Package main ...
package main

import (
	"github.com/go-rod/rod"
	"github.com/go-rod/rod/lib/proto"
)

func main() {
	browser := rod.New().MustConnect()

	page := browser.MustPage("http://example.com")

	err := proto.AuditsEnable{}.Call(page)
	if err != nil {
		panic(err)
	}
}
```

<img width="1231" alt="image" src="https://user-images.githubusercontent.com/1415488/192467726-038efe6c-fd0f-4d45-a37d-40c5823553be.png">

**Reply by freb** [reply]
Thanks so much. I was using client directly instead of page.

**Reply by ysmood** [reply]
Then you can check how `rod.Page` implements the `proto.Client` interface.
